docs/src/references/diagnostic-plugin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Diagnostic plugin
+tags: [Reference, API]
+sidebar_position: 100
+description: Diagnostic Plugin API reference
+---
+
+# Contract between runner and plugins
+
+## Runner's responsibility
+* Runner creates an output directory `<PLUGIN_DIR>/<TARBALL_NAME>` and subdirectories for each plugin `<PLUGIN_DIR>/<TARBALL_NAME>/<PLUGIN_NAME>`.
+* Runner doesn't know about each plugin script. Runner executes all the plugin scripts under `<PLUGIN_DIR>` blindly with arguments.
+    * arg1: `collect`
+    * option1 `--output-dir` <PATH>: the path to the subdirectory, `<PLUGIN_DIR>/<TARBALL_NAME>/<PLUGIN_NAME>`
+    * option2 `--config-dir` <PATH>
```

Comment:
Using `<PATH>` confuses docusaurus which expects then some HTML.

```suggestion
    * option1 `--output-dir`  $DIR_PATH: the path to the subdirectory, `<PLUGIN_DIR>/<TARBALL_NAME>/<PLUGIN_NAME>`
    * option2 `--config-dir`  $DIR_PATH
```

Reply:
After I rewrote the file, the error seems gone :+1: